OasisDEX/oasis-borrow
# Ades/new positions overview header

Updates positions styles to be:

<img width="1240" alt="image" src="https://user-images.githubusercontent.com/3783198/174045670-9712dbf0-08d6-457a-a683-e83899cf2aa0.png">

Fixes some other style things from https://www.notion.so/oazo/Earn-release-feedback-f994cdb26e9e42c1a0bfa577350f55c1.

How to test:

- Few different browsers including mobile.
- Ensure that assets and positions actions can still be performed.
- Check positions take in to account the debt.
